drivers/clock_control/clock_control_mcux_syscon.c

```diff
@@ -45,6 +45,28 @@ static int mcux_lpc_syscon_clock_control_on(const struct device *dev,
 
 #if defined(CONFIG_PINCTRL_NXP_KINETIS)
 	switch ((uint32_t)sub_system) {
+#if defined(CONFIG_SOC_SERIES_MCXA)
+	case MCUX_PORT0_CLK:
+		CLOCK_EnableClock(kCLOCK_GatePORT0);
+		RESET_ReleasePeripheralReset(kPORT0_RST_SHIFT_RSTn);
```

Comment:
Reset signals should not be handled from the clock control driver. Unfortunately, the best way to handle this is likely to do the following:
- implement a proper reset driver for the MRCC peripheral
- call `reset_line_toggle` from the GPIO driver initialization routine

If you'd like an example for reset drivers, please look here: #72120. I'm aware this is a significant scope creep for enabling this SOC, so as an alternative I am ok if we clear these reset signals in an SOC init function registered with `SYS_INIT` for the time being. I'd prefer we just get the reset driver implemented though